### PR TITLE
Croteau hw01

### PR DESCRIPTION
**Pull request by sabreena-croteau**
Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

### CODE REVIEW

**Comment by sabreena-croteau** [comment]
Hi!

This was a bit hard because I have never used the shell or terminal before. Or, at least it was hard until I felt like I got the hang of it. As you can see, there's only one commit where I actually changed the file, and the rest are just trial attempts when I was trying to figure it out. If there is something that is way off base, please let me know so I can learn it correctly. Thank you!

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+
+I have completed my two comprehensive exams in substantive field (International relations as primary and comparative political as secondary) and am working towards the Social Sciences certificate in Quantitaive Methods. Most of the methodology classes I have taken assumed coding ability and concentrated heavily of mathematical concepts. While I feel strong in my understanding of the methods, my coding is quite weak and frequently caused me to struggle with the assignments for these courses (I knew what I wanted to do but could not figure out how to tell it to the computer). For these reasons, some part of this class will be repetitive for me, but I feel it is necessary to return to the building blocks, especially since coding in R does not come easy nor do I enjoy the puzzle of trying to figure it out (which is true when I am learning the mathematical side of working with data). But, I have heard very good things about the course from others who have taken it, so I am confident it will help me get back on track.
```
Comment (wanitchayap) [inline README.md:16]: I can second on the good things about this course! It is one of the best course (and the most useful) I have taken here :)

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+#### **How I Did This**
+* Followed all the instructions on the course website. But, though the intructions were very clear, I didn't really understand what I was doing, just blindly following along
+* So I had a friend that uses Git everyday walk through everything with me to explain everything, this was very helpful.
+* Once I understood how to push and pull from Git just with the shell, I gave it a try in RStudio
+* I also did find the [reference page](https://cfss.uchicago.edu/faq/homework-guidelines/#homework-workflow) to be very helpful
+* This little [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) also helped as a reference
+
+
+Well, there it is! My first pull request homework assignment attempt. Hope you like it!
+![](http://media.tumblr.com/tumblr_m22kl3NyNA1r6oqyh.gif)
```
Comment (wanitchayap) [inline README.md:27]: **Thank you** for your amazing work too! Also great GIF 😉

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |      X     |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |    X       |              |            |
| **Reproducibility**         |    X       |              |            |

Remarks:
* Very comprehensive bio and you have done more than we require. Great job!
* Frequent commits with detail commit messages 👍 
* For future reference, you don't need to commit the `.html`, `.gitignore`, and `.RProj`

**Comment by Wendychenn** [comment]
Hi Croteau,
  You've done a good job on this HW01! I think your introduction is really impressive, because you're not just talking about your past but also the reason why you choose this course and your expectation from it which is great. I also like the way you insert you list into the file. You represent the list in a logical and smooth way that is easy to get into. Finally, I want to say your arrangement is beautiful and easy to read.
  I find that the difference between two commits is not so huge, so I presume you almost completed all your work and pushed them to the GitHub directly. I think it will be very beneficial if you can push your work several times when you’re working on it, because if your software goes crash, all your works might be missing if you didn’t upload them frequently. And I don’t know why you put the link in the middle of your introduction because I don’t see your expiation. Finally, I cannot open the link so it’s better to check you content after you finish you work!

 Wen
 
## Evaluation 

Topic | Excellent | Satisfactory | Needs Improvement
-- | -- | -- | --
Coding style |   | ⭕ |  
Coding strategy | ⭕ |  |  
Presentation |  |  ⭕ |  
Achievement, creativity | ⭕ |   |  
Reproducibility | ⭕ |   |

**Review by Wendychenn — COMMENTED** [review]
Hi Croteau,
  You've done a good job on this HW01! I think your introduction is really impressive, because you're not just talking about your past but also the reason why you choose this course and your expectation from it which is great. I also like the way you insert you list into the file. You represent the list in a logical and smooth way that is easy to get into. Finally, I want to say your arrangement is beautiful and easy to read.
  I find that the difference between two commits is not so huge, so I presume you almost completed all your work and pushed them to the GitHub directly. I think it will be very beneficial if you can push your work several times when you’re working on it, because if your software goes crash, all your works might be missing if you didn’t upload them frequently. And I don’t know why you put the link in the middle of your introduction because I don’t see your explanation. Finally, I cannot open the link so it’s better to check your content after you finish!

 Wen
 
## Evaluation 

Topic | Excellent | Satisfactory | Needs Improvement
-- | -- | -- | --
Coding style |   | ⭕ |  
Coding strategy | ⭕ |  |  
Presentation |  |  ⭕ |  
Achievement, creativity | ⭕ |   |  
Reproducibility | ⭕ |   |

**Review by noahttesfaye — COMMENTED** [review]
I really enjoyed reading your bio! I thought that text and story aside you were able to code and format it in a really simple way. You didn't complicate any of the text formats and used them in a way that made it really easy for me to follow. If there is even anything I would maybe suggest, you could integrate the first hyperlink into your first paragraph. But other than that, I really enjoyed learning about your interests in international security.

**Comment by cweis22** [comment]
Hi Sabrina,

Great job overall on the assignment! I am very impressed with your clear effort into understanding the material, and into creating an honest and frank bio. Here is some moretechnical feedback:

**Three things that you did well:**
1) I like the combination of bold and italics in the "About Me" heading.
2) I like the use of links within the text for the "cheatsheet" and "reference page" references.
3) Great job adding not just an image, but a gif! I had no idea that adding the two would work in the same way, but. now that Iknow it seems rather obvious.

**Three things to improve:**
1) It looks like the loink to (what I presume is your) dissertation has expired. Of course, this is a problem with the link itself, and not the code, but it would be a shame for it to happen in a higher-stakes document, so I'd recommend making sure that it doesn't happen again!
2) Regarding tht same link, it would have been great to have incorporated it into your text, as you did with the other two.
3) Having a picture of yourself would have completed this bio nicely!

Again, great job overall, and best of luck with the class!

**Pull request closed without merging**